frontend/src/pages/CalendarPage/index.js

```diff
@@ -138,36 +150,41 @@ class CalendarPage extends React.Component {
           </Grid>
 
           <Grid item>
-            {OfficerRenderPermission(FormControlLabel)({
-              control: (
-                <Checkbox
-                  name='pending'
-                  onChange={() => this.handlePendingChange()}
-                  checked={pending}
-                />
-              ),
-              label: 'Pending',
-            })}
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name='ready'
-                  onChange={() => this.handleReadyChange()}
-                  checked={ready}
-                />
-              }
-              label='Ready'
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name='complete'
-                  onChange={() => this.handleCompleteChange()}
-                  checked={complete}
-                />
-              }
-              label='Complete'
-            />
+            {view === 'calendar' &&
```

Comment:
For this block could just have 1 ternary statement? view==='calendar' can just control all of the 3 FormControlLabels.